src/components/order/StatusLabel/index.tsx

```diff
@@ -1,12 +1,37 @@
 import React from 'react'
-import styled from 'styled-components'
+import styled, { DefaultTheme } from 'styled-components'
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
 import { faCheckCircle, faClock, faDotCircle, IconDefinition } from '@fortawesome/free-solid-svg-icons'
 
 import { OrderStatus } from 'api/operator'
 
 export type Props = { status: OrderStatus }
 
+function setStatusColors({ theme, status }: { theme: DefaultTheme; status: OrderStatus }): string {
+  let background, text
+
+  switch (status) {
+    case 'expired':
+      text = theme.labelTextExpired
```

Comment:
why not `color`? haha super nit time

Reply:
Fun story, that's what I had.
But both are colors (background and text) to I would either prefix both or none. I chose to keep only the name of the css property.